wgsl/index.bs

```diff
@@ -9770,7 +9815,13 @@ Most built-in functions have tags of:
 
 Here is the list of exceptions:
 - All functions in [[#sync-builtin-functions]] have a [=call site tag=] of  [=CallSiteRequiredToBeUniform=].
-- All functions in [[#derivative-builtin-functions]], [[#texturesample]], [[#texturesamplebias]], and [[#texturesamplecompare]] have a [=call site tag=] of [=CallSiteRequiredToBeUniform=] and a [=function tag=] of [=ReturnValueMayBeNonUniform=].
+- All functions in
+    [[#derivative-builtin-functions]], [[#texturesample]], [[#texturesamplebias]], and [[#texturesamplecompare]]:
+    - have a [=function tag=] of [=ReturnValueMayBeNonUniform=].
+    - have a [=call site tag=] as follows:
+          - [=CallSiteNoRestriction=] if the [=option/fragmentShaderDerivativesCheckedUniformity=] option is set to `false`,
+               or if the call site has an [=attribute/unchecked_uniformity=] attribute.
```

Comment:
This special-casing is unnecessary; you've already handled the `@unchecked_uniformity` case when you sever the edge from `RequiredToBeUniform` in the changes you made to function calls (and it makes more sense to handle it there anyway).

Reply:
removed. thanks